<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Player</class>
 <widget class="QMainWindow" name="Player">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Player</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>320</width>
      <height>200</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="userPage">
     <widget class="QListWidget" name="userList">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>320</width>
        <height>200</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>200</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="autoScroll">
       <bool>false</bool>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="isWrapping" stdset="0">
       <bool>false</bool>
      </property>
      <property name="gridSize">
       <size>
        <width>3</width>
        <height>1</height>
       </size>
      </property>
      <property name="viewMode">
       <enum>QListView::ListMode</enum>
      </property>
      <property name="uniformItemSizes">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>Andy</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rosemary</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Eliot</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="playerPage">
     <widget class="QLabel" name="albumTextLabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>180</y>
        <width>180</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Current Song Info</string>
      </property>
     </widget>
     <widget class="QPushButton" name="browseLeft">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>70</width>
        <height>200</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>500</height>
       </size>
      </property>
      <property name="text">
       <string>Prev
Album</string>
      </property>
     </widget>
     <widget class="QLabel" name="albumImageLabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>0</y>
        <width>180</width>
        <height>180</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="text">
       <string>Current Album Info</string>
      </property>
     </widget>
     <widget class="QPushButton" name="browseRight">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>0</y>
        <width>70</width>
        <height>200</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="text">
       <string>Next
Album</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="controlWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>320</width>
      <height>40</height>
     </rect>
    </property>
    <widget class="QPushButton" name="nextButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>3</y>
       <width>45</width>
       <height>35</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>45</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>Next
Song</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stopButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>3</y>
       <width>45</width>
       <height>35</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>45</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="menuButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>3</y>
       <width>45</width>
       <height>35</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>45</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>Menu</string>
     </property>
    </widget>
    <widget class="QPushButton" name="userButton">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>3</y>
       <width>45</width>
       <height>35</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>45</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>Users</string>
     </property>
    </widget>
    <widget class="QPushButton" name="playPauseButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>3</y>
       <width>45</width>
       <height>35</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>45</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>Play/
Pause</string>
     </property>
    </widget>
    <widget class="QPushButton" name="prevButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>3</y>
       <width>45</width>
       <height>35</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>45</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>Prev
Song</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
